internal/databases/postgres/backup_push_handler.go

```diff
@@ -652,6 +655,30 @@ func (bh *BackupHandler) initBackupTerminator() {
 	}()
 }
 
+func (bh *BackupHandler) checkDataChecksums() error {
+	if bh.Arguments.verifyPageChecksums {
+		tracelog.DebugLogger.Println("checkDataChecksums: Checking data_checksums setting.")
+
+		dataChecksums, err := bh.Workers.QueryRunner.GetDataChecksums()
+		if err != nil {
+			return err
+		}
+
+		if dataChecksums != "on" {
+			tracelog.WarningLogger.Println(
```

Comment:
just warning, why don't we reset  bh.Arguments.verifyPageChecksums?

Reply:
> just warning, why don't we reset bh.Arguments.verifyPageChecksums?

fixed (edc052721d15cef0fea6c2542b919c02b25e3b60)

Reply:
Hi @x4m  ⬆️